### DISCUSSION TITLE
Link not working

### DISCUSSION THREAD

**Discussion by Tiger2Henschel** [Official-Husko/NN-Downloader#33] [Q&A] [answered]
The link to https://multporn.net/comics/what_didnt_happen doesn't work and it says the link is invalid.

**Comment by Official-Husko** [comment]
Yep this link somehow does not work. Investigating it.

Also please open an issue next time instead of a discussion. Further replies will be in the created issue #34.

**Reply by Official-Husko** [reply, marked as answer]
Alright issue has been resolved in version [1.6.3](https://github.com/Official-Husko/NN-Downloader/releases/tag/1.6.3).

**Reply by Tiger2Henschel** [reply]
Thanks
